worldedit-core/src/main/java/com/sk89q/worldedit/command/tool/AreaPickaxe.java

```diff
@@ -62,29 +62,29 @@ public boolean actPrimary(Platform server, LocalConfiguration config, Player pla
             return true;
         }
 
-        EditSession editSession = session.createEditSession(player);
-        editSession.getSurvivalExtent().setToolUse(config.superPickaxeManyDrop);
+        try (EditSession editSession = session.createEditSession(player)) {
```

Comment:
It may be worth moving the current stuff here into the other try block just so we don't have two